### DISCUSSION TITLE
수치해석 프로젝트 커리큘럼 문의

### DISCUSSION THREAD

**Discussion by hyoseok-bang** [Pseudo-Lab/9th-builder#33] [Issue and Q&A]
안녕하세요,

수치해석 프로젝트 진행 방식이 궁금하여 문의드립니다.

안내를 보니 중간/기말 고사 얘기도 나오고 인하대 학생들이 많다는 말도 있고… 학교에서 오프라인으로 듣는 수업인가요??

자세한 진행 방식을 알려주시면 감사하겠습니다

**Comment by seriousran** [comment]
@imrealhelper 님! 답변해주시면 좋을 것 같습니다! :)

**Comment by imrealhelper** [comment]
안녕하세요. 빌더 임진우입니다.

현재 인하대학교에서는 수치해석 프로젝트라는 과목이 오프라인으로 진행되고 있습니다.

하지만 스터디에서는 오프라인 강의가 아닌, 매주 지정된 온라인 강의를 듣고 정리하는 방식으로 진행됩니다.

스터디는 주 발표자와 보조 발표자로 나누어 진행됩니다.

주 발표자는 강의를 듣고 내용을 요약하여 PPT로 발표합니다.

보조 발표자는 매주 정해진 과제를 풀고, 개념이 어떻게 사용되었는지와 문제 접근 방법에 대해 발표합니다.

또한, 인하대학교 학생이 아니어도 스터디에 잘 참여하실 수 있도록 준비되어 있으니 걱정하지 않으셔도 됩니다 :)

**Reply by hyoseok-bang** [reply]
답변 감사드립니다!
